### PR TITLE
Add layer's extent button for WMTS GetCap item

### PR DESCRIPTION
**Pull request by oterral**

### CODE REVIEW

**Comment by oterral** [comment]
Failure not related to the PR I think

File: src/modules/import/wmtsgetcap.js [hunk 1]
```diff
@@ -14,10 +14,31 @@ goog.require('ol.source.WMTS');
...
+  // Get the layer extent defines in the GetCapabilities
+  const getLayerExtentFromGetCap = function(getCapLayer, proj) {
+    const wgs84 = 'EPSG:4326';
+    const layer = getCapLayer;
+    const projCode = proj.getCode();
+    const wgs84Extent = layer['WGS84BoundingBox'];
+    if (wgs84Extent) {
+      // If only an extent in wgs 84 is available, we use the
+      // intersection between proj extent and layer extent as the new
+      // layer extent. We compare extients in wgs 84 to avoid
```
Comment (gberaudo) [inline src/modules/import/wmtsgetcap.js]: typo

**Comment by gberaudo** [comment]
@oterral, I updated the branch to trigger a travis build.

**Comment by oterral** [comment]
@gberaudo Typo fixed

**Comment by fredj** [comment]
travis timeout, restarted

**Comment by gberaudo** [comment]
It is passing, merging.

**Pull request merged**